lib/test_track.rb

```diff
@@ -21,16 +21,32 @@ module TestTrack
 
   class << self
     def analytics
-      @analytics ||= wrapper(mixpanel)
+      @analytics ||= analytics_wrapper(mixpanel)
     end
 
     def analytics=(client)
-      @analytics = client.is_a?(TestTrack::Analytics::SafeWrapper) ? client : wrapper(client)
+      @analytics = client.is_a?(TestTrack::Analytics::SafeWrapper) ? client : analytics_wrapper(client)
+    end
+
+    def misconfiguration_notifier
+      @misconfiguration_notifier ||= MisconfigurationNotifier::Wrapper.new(default_notifier)
+    end
+
+    def misconfiguration_notifier=(notifier)
+      @misconfiguration_notifier = MisconfigurationNotifier::Wrapper.new(notifier)
```

Comment:
Is the only kind of explicit Errbit-style notifying we do in MisconfigurationNotifier? Everything else we just figure you've wired in an exception alerting tool somewhere up stack?

Reply:
Yea, grepping for Airbrake only turned up this file and its specs